tests/kafkatest/tests/core/downgrade_test.py

```diff
@@ -67,11 +67,18 @@ def setup_services(self, kafka_version, compression_types, security_protocol):
                              version=kafka_version)
         self.producer.start()
 
+        static_membership = kafka_version == DEV_BRANCH or kafka_version >= LATEST_2_3
```

Comment:
Why does static membership make a difference here?

Reply:
Good question, without it the verifiable consumer fails with:
```
usage: verifiable-consumer [-h]                                                                                                                                                                                                                              
                           --broker-list HOST1:PORT1[,HOST2:PORT2[...]]                                                                                                                                                                                      
                           --topic TOPIC --group-id GROUP_ID                                                                                                                                                                                                 
                           --group-instance-id GROUP_INSTANCE_ID                                                                                                                                                                                             
                           [--max-messages MAX-MESSAGES]                                                                                                                                                                                                     
                           [--session-timeout TIMEOUT_MS] [--verbose]                                                                                                                                                                                        
                           [--enable-autocommit]                                                                                                                                                                                                             
                           [--reset-policy RESETPOLICY]                                                                                                                                                                                                      
                           [--assignment-strategy ASSIGNMENTSTRATEGY]                                                                                                                                                                                        
                           [--consumer.config CONFIG_FILE]                                                                                                                                                                                                   
verifiable-consumer: error: argument --group-instance-id is required                                                                                                                                                                                         
```

I assumed that this check was added for a reason, however maybe it's being to strict?

Reply:
I believe it's due to 2.4 accidentally requiring `--group-instance-id` and in this test we end up using this version. See 0f995ba6be1c0f949320b0879241d9a7c9578436

Reply:
I could force static throttling to be used when the version is 2.4 or 2.3 and leave a comment if that works for you.

Reply:
@abbccdda I changed the code to add a static_membership parameter to the test matrix. We simply avoid this for 2.4 where this is broken due to the command line validation in that release.

Reply:
@abbccdda friendly ping on this

Reply:
I see, makes sense.